lib/krane/kubectl.rb

```diff
@@ -34,7 +34,16 @@ def run(*args, log_failure: nil, use_context: true, use_namespace: true, output:
       (1..attempts).to_a.each do |current_attempt|
         logger.debug("Running command (attempt #{current_attempt}): #{cmd.join(' ')}")
         out, err, st = Open3.capture3(*cmd)
-        logger.debug("Kubectl out: " + out.gsub(/\s+/, ' ')) unless output_is_sensitive
+
+        # https://github.com/Shopify/krane/issues/395
+        if out.encoding != Encoding::UTF_8
```

Comment:
> it seems like in other cases (e.g. utf-16) this might be wrong?

What are you thinking would cause the string we get back from kubectl to actually have a different encoding?